nssp_demo/postprocess_state_forecast.R

```diff
@@ -121,28 +139,25 @@ make_one_forecast_fig <- function(target_disease,
       hjust = "left",
       vjust = "bottom",
     ) +
-    ggtitle(title, subtitle = glue("as of {last_data_date}")) +
+    ggtitle(title, subtitle = glue("as of {data_vintage_date}")) +
     y_scale +
     scale_x_date("Date") +
-    scale_shape_discrete("Data Type", labels = str_to_title) +
-    scale_fill_brewer(
-      name = "Credible Interval Width",
-      labels = ~ percent(as.numeric(.))
-    ) +
+    # scale_shape_discrete("Data Type", labels = str_to_title) +
     theme(legend.position = "bottom")
 }
 
 
-make_forecast_figs <- function(model_run_dir,
-                               filter_bad_chains = TRUE,
-                               good_chain_tol = 2) {
+postprocess_state_forecast <- function(model_run_dir,
+                                       filter_bad_chains = TRUE,
+                                       good_chain_tol = 2) {
   state_abb <- model_run_dir |>
     path_split() |>
     pluck(1) |>
     tail(1)
 
-  data_path <- path(model_run_dir, "data", ext = "csv")
-  inference_data_path <- path(model_run_dir, "inference_data",
+  train_data_path <- path(model_run_dir, "data", ext = "csv")
+  eval_data_path <- path(model_run_dir, "eval_data", ext = "tsv")
+  inference_train_data_path <- path(model_run_dir, "inference_data",
```

Comment:
```suggestion
  inference_data_path <- path(model_run_dir, "inference_data",
```

Change throughout. Must have been a find and replace run amok

Reply:
Fixed throughout in 9b0b2e9